Makefile

```diff
@@ -202,8 +202,8 @@ $(BINDIR):
 
 # Util targets
 ##############
-.PHONY: verify verify-generated verify-client-gen verify-docs
-verify: .init verify-generated verify-client-gen verify-docs verify-vendor
+.PHONY: verify verify-generated verify-client-gen
+verify: .init verify-generated verify-client-gen verify-vendor
```

Comment:
why do we want to take verify-docs out of verify-everything?

Reply:
That's was a mistake, I thought I reverted that change, let me try again.